README.rst

```diff
@@ -1,39 +1,40 @@
-The Chrome Extension for codesy.io
-==================================
-The codesy.io Chrome Extension adds codesy.io information to open-source bugs
+The codesy.io widget
+====================
+The codesy.io widget is Chrome Extension and Firefox Addon the adds codesy.io information to open-source bugs
 you visit.
 
 
-To use the extension 'unpacked'
--------------------------------
-1. Clone this repo
-2. Copy the prod/manifest.json into the root of the project
-   (You can also run gulp dev-start)   
-3. Follow these instructions (http://developer.chrome.com/extensions/getstarted.html#unpacked)
-
-
 Development
 -----------
 1. Clone this repo
 2. Set up Gulp
-
    * node install gulp
    * 'npm install' to install dependencies
+3. For firefox run 'npm install jpm'
+
+
+To use the 'unpacked' Chrome Extension
+--------------------------------------
+1. Run 'gulp dev-chrome'   
+2. Follow these instructions (http://developer.chrome.com/extensions/getstarted.html#unpacked) 
+3. Point to the unpacked codesy chrome extension in ./chrome.
+4. Reload the extension page (chrome://extensions/)
+
+
+To use the Firefox Addon
+------------------------
+1. Run 'gulp dev-firefox'   
+2. Run 'jpm run --profile default'  (this runs a new instance of firefox with your default profile so your bookmarks, etc. are available)
```

Comment:
You have to run `jpm` from the `firefox` directory. So this should be:

1. `gulp dev-firefox`
2. `cd firefox`
3. `jpm run --profile default`